src/components/protocol_handler/include/protocol_handler/incoming_data_handler.h

```diff
@@ -67,7 +67,7 @@ class IncomingDataHandler {
    * \return list of complete, correct packets
    */
   ProtocolFramePtrList ProcessData(const RawMessage& tm_message,
-                                   RESULT_CODE* result,
+                                   RESULT_CODE& result,
```

Comment:
@SKobziev rename result to out_result. It would be more obvious that is ouptut parameter
Also specify it explicitly in description